open_event/static/admin/js/scheduler.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * Created by Niranjan on 23-May-16.
+ */
+(function ($) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l4)